src/data/objects/ipaddress.yaml

```diff
@@ -31,11 +31,14 @@ schema:
     type: String
     value: 255.255.255.0
     description: The subnet mask.
+  public:
+    type: Boolean
+    value: true
+    description: Whether this is a public of private IP address.
   type:
-    type: Enum
-    subtype: IPAddressType
-    value: public
-    description: The type of IP Address, either public or private.
+    type: String
```

Comment:
This could probably still be an Enum of `ipv4`, `ipv6`, `ipv6/range`, and `ipv6/pool`, but I don't feel that strongly about it.